__tests__/core/Vector_test.re

```diff
@@ -0,0 +1,221 @@
+open Jest;
+
+let it = test;
+
+describe("Vector", () => {
+  // Test constants.
+  let v1Int: Vector.t(int) = (5, 6);
+  let v2Int: Vector.t(int) = (7, (-2));
+
+  let v1Float: Vector.t(float) = (5.4, 6.8);
+  let v2Float: Vector.t(float) = (7.2, (-2.6));
+
+  let sInt: int = 8;
+  let sFloat: float = 8.25;
+
+  let roundness: float = 0.5;
+
+  // Begin tests.
+  describe("add", () =>
+    it(
+      "should expose a function for adding two vectors composed of integer values",
+      () =>
+      Expect.(expect(Vector.add(~v1=v1Int, ~v2=v2Int)) |> toEqual((12, 4)))
+    )
+  );
+
+  describe("addf", () => {
+    it(
+      "should expose a function for adding two vectors composed of float values (x value)",
+      () =>
+      Expect.(
+        expect(Vector.addf(~v1=v1Float, ~v2=v2Float) |> fst)
+        |> toBeCloseTo(12.6)
+      )
+    );
+
+    it(
+      "should expose a function for adding two vectors composed of float values (y value)",
+      () =>
+      Expect.(
+        expect(Vector.addf(~v1=v1Float, ~v2=v2Float) |> snd)
+        |> toBeCloseTo(4.2)
+      )
+    );
+  });
```

Comment:
Here's an example of what I was referencing in the PR description – `Vector.addf` returns a tuple of floats, `(float, float)`. However, we can't just use `toEqual((12.6, 4.2))` due to floating point precision issues. So I have two separate tests, one for the x value with `toBeCloseTo` and one for the y value, also with `toBeCloseTo`. Curious to hear about other approaches here, given that `bs-jest` only allows one assertion per test. Also curious on folks' thought re: `toBeCloseTo` or `toBeSoCloseTo`. I used both in these tests, opting for the latter in cases where significant figures seemed to be important, but this is pretty arbitrary.